labels.md

```diff
@@ -0,0 +1,220 @@
+# Goals
+
+* synchronise labels between repos
```

Comment:
do we need to capitalize the first letter?
/cc @dcalvin

Reply:
We should - but it does not matter here, as the point of this document is to finalize the labels.